src/doom/p_sight.c

```diff
@@ -40,6 +41,44 @@ fixed_t		t2y;
 int		sightcounts[2];
 
 
+// PTR_SightTraverse() for Doom 1.2 sight calculations
+// taken from prboom-plus/src/p_sight.c:69-102
+boolean PTR_SightTraverse(intercept_t *in)
+{
+  line_t *li;
```

Comment:
Please fix formatting/indentation to match the rest of the file. Read `HACKING` in the top level for style guidelines.

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1205/commits/77f49e5ea5aae45fe904306a9364018c0fc23b71 .